components/popover/index.en-US.md

```diff
@@ -0,0 +1,29 @@
+---
+category: Components
+chinese: 气泡卡片
+type: Views
+english: Popover
+---
+
+The bubble float card popped by clicking or hovering.   
+
+## When to use
+
+For more information or operations, put all into the popover card to show according to the behavior.
+
+The difference with `Tooltip` is the popover card can provide operations and much more complicated information such as link and button.
```

Comment:
Comparing with `Tooltip`, besides information `Popover` card can also provide action elements like links and buttons.